app/scripts/lib/auth-errors.js

```diff
@@ -187,6 +187,10 @@ function (_, Errors) {
     INVALID_EMAIL: {
       errno: 1023,
       message: t('Valid email required')
+    },
+    FORCE_AUTH_EMAIL_REQUIRED: {
```

Comment:
> Any details on the noise you're seeing?

I'm used to watching the unit test console, which is very noisy. The main app, not so much.